src/tools/speak_to_agent/definition.json

```diff
@@ -0,0 +1,32 @@
+{
+    "name": "speak_to_agent",
+    "description": "Sends a message to a previously spawned agent for follow-up instructions or status updates",
```

Comment:
should be more detailed, encourage to base behaviour on earlier get_ageents, and return_results also explaining how agent status affects if message should be send or not.